### ISSUE TITLE
Does this work?

### ISSUE THREAD

**Issue by mflodzinski** [msalhab96/RNN-Transducer#3]
Does this code even work? Apart from the fact that "log()" method in Loss class is gone and I had to replace it with "torch.log()", the main  loss function almost always returns "nan" values thus the model is not learning at all.

I would be grateful to here back from the author :)

**Comment by reddyn12** [comment]
@msalhab96 Is the loss function correct? Isn't the backwards run supposed to be custom?
